server/helpers/app_helpers.py

```diff
@@ -0,0 +1,9 @@
+import os
+
+
+def is_production() -> bool:
+
+    if os.getenv('PYTHON_ENV') == 'production':
```

Comment:
`return os.getenv('PYTHON_ENV') == 'production'`

Reply:
The concise vs readable debate! Will go with your suggestion, doesn't require my experience and is simple so with you on this one